js/numbers.js

```diff
@@ -0,0 +1,46 @@
+function check1()
+{
+    var number = document.getElementById("field").value;
```

Comment:
Read about DRY pattern. You are repeating the code in many places, for example you can define separated function `render` for this: `document.getElementById("score").innerHTML`